src/locales/fr/PAGES/SUMMARY.json

```
@@ -0,0 +1,19 @@
+{
+    "TITLE": "Étape 5: Résultat du rapport de l’évaluation",
+    "INTRO": "Fournir des informations supplémentaires sur l’évaluation que vous souhaitez voir figurer dans le rapport. Les résultats de l’évaluation que vous avez saisis à l’étape précédente sont inscrits ci-dessous dans la section « Résultats détaillés de l’audit » pour examen et pour vous aider à rédiger le résumé.",
+    "RESOURCE_LINK_NAME": "Étape 5: Résultat du rapport de l’évaluation",
+    "LABEL_TITLE": "Titre du rapport",
+    "TITLE_PREFIX": "Rapport pour",
+    "INF_TITLE": "Donnez un titre au rapport d’évaluation de l’accessibilité du site Web. Par exemple : <br>« Rapport d’évaluation pour une organisation donnée »<br> « Analyse de l’accessibilité de la boutique en ligne et suggestions d’amélioration ».",
+    "LABEL_COMMISSIONER": "Commandé par",
+    "INF_COMMISSIONER": "La personne, l’équipe, l’organisation, le service interne ou toute autre entité qui a commandé l’évaluation.",
+    "LABEL_CREATOR": "Évaluateur",
+    "INF_CREATOR": "La personne, l’équipe, l’organisation, le service interne ou toute autre entité responsable de la réalisation de l’évaluation.",
+    "LABEL_DATE": "Date de l’évaluation",
+    "INF_DATE": "Indiquer la date d’achèvement ou la période de cette évaluation. Vous pouvez utiliser n’importe quel format de date. Écrire le mois en toutes lettres pour que ce soit clair au niveau international. (Le format de date 01-02-2022 est ambigu - il s’agit du 2 janvier dans certains endroits et du 1er février dans d’autres).",
```

```suggestion
    "INF_DATE": "Indiquez la date d’achèvement ou la période de cette évaluation. Vous pouvez utiliser n’importe quel format de date. Écrivez le mois en toutes lettres pour qu'il soit compréhensible internationalement. (Le format de date 01-02-2022 est ambigu – il s’agit du 2 janvier dans certains endroits et du 1er février dans d’autres).",
```

See previous remark on use of infinitive + suggestion.